src/components/Popup.jsx

```diff
@@ -7,25 +7,25 @@ class Popup extends React.Component {
     this.popupBody = null;
     this.close = this.close.bind(this);
   }
-  
+
   render() {
     return (
       <div className="popup" ref={(c)=>{this.popupBody=c}} onClick={(e) => { return e.target === this.popupBody && this.close(e); }}>
         <div className="popup-title">
-          <button className="fa fa-close" onClick={this.close}></button>
+          <i className="fa fa-close" onClick={this.close}></i>
```

Comment:
Maybe set up a global style for `button` that resets it to something that fits the design, then override or add to that for specific buttons.